### PR TITLE
Added third party cookies test component in teams test app

### PR DESCRIPTION
**Pull request by Ella-ly**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

Added third party test component in teams test app. This component is for validating `allow-storage-access-by-user-activation` attribute which allows app develops to use storage access API (Web API). 

### Main changes in the PR:

1. Added third party cookies test component in teams test app.

### Change file added:

No change files are needed.

### Screenshots:
image

image

### CODE REVIEW

File: apps/teams-test-app/src/components/ThirdPatryCookies.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    onClick: refreshCookies,
+  });
+};
+
+const hasCookieAccess = async (): Promise<boolean> => {
+  if (!document.requestStorageAccess) {
+    console.log(
+      'Storage Access API not supported. Assume it is an older browser that doesn not block 3P cookies and we have access.',
+    );
+    return true;
```
Comment (maglims) [inline apps/teams-test-app/src/components/ThirdPatryCookies.tsx:55]: oh, is it supposed to return true on storage access api not supported?

File: apps/teams-test-app/src/App.tsx [hunk 2]
```diff
@@ -182,6 +183,7 @@ const App = (): ReactElement => {
         <VideoAPIs />
         <VideoExAPIs />
         <VisualMediaAPIs />
+        <CookieAccessComponent />
```
Comment (MengyiGong) [inline apps/teams-test-app/src/App.tsx:186]: It would be great to follow the alphabetical order.

Comment (Ella-ly) [reply apps/teams-test-app/src/App.tsx]: Updated.

File: apps/teams-test-app/src/components/ThirdPatryCookies.tsx (hunk 1 above)
Comment (Ella-ly) [reply apps/teams-test-app/src/components/ThirdPatryCookies.tsx:55]: Basically, if it is an older browser that does not block 3P cookies then we will have access and do not need to call Storage Access APIs to request access. We can use and set the document.cookie directly, for example, `document.cookie = 'testCookies=testvalue; path=/; Secure; SameSite=None';`. Please see the [Checking and requesting storage access](https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API/Using#checking_and_requesting_storage_access) section in Using the Storage Access API document. I simplified the code to better align with the test need.

**Review by MengyiGong — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**